core/src/main/scala/cats/derived/functor.scala

```diff
@@ -52,6 +52,15 @@ private[derived] abstract class MkFunctorNested extends MkFunctorCons {
       def safeMap[A, B](fa: F[A])(f: A => Eval[B]) =
         mkSafeMap(F.fo)(F.unpack(fa))(mkSafeMap(F.fi)(_)(f)).map(F.pack)
     }
+
+  implicit def mkFunctorFromContraNested[F[_]](implicit F: Split1[F, Contravariant, Contravariant]): MkFunctor[F] =
```

Comment:
Would mind adding a test for this? Something like:
```scala
type Pred[-A] = A => Boolean
Functor[Lambda[a => Pred[Pred[a]]]]
```

I'm just worried about implicit resolution. It looks like it should be ambiguous with the `mkFunctorNested` method :thinking:

Reply:
You know, I've been playing with that and I'm not sure it's quite right, actually. I'll mess with the tests a bit and make sure that it works correctly.